docs/ldml/tr35-numbers.md

```diff
@@ -1315,12 +1319,14 @@ For the formatting of number ranges, see <a href="#Number_Range_Formatting">Numb
 
 ## <a name="Rule-Based_Number_Formatting" href="#Rule-Based_Number_Formatting">Rule-Based Number Formatting</a>
 
-```xml
+```dtd
 <!ELEMENT rbnf ( alias | rulesetGrouping*) >
 
-<!ELEMENT rulesetGrouping ( alias | ruleset*) >
+<!ELEMENT rulesetGrouping ( alias | rbnfRules? ruleset*) >
 <!ATTLIST rulesetGrouping type NMTOKEN #REQUIRED>
 
+<!ELEMENT rbnfRules ( #PCDATA )>
+
```

Comment:
This is the key change to the spec. All of the following documentation involving RBNF is just to describe the syntax of `<rbnfRules>`, which is severely under-documented.